File: Data Structures & Algorithms/is-anagram/submission-11.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        vector<int>v(26);
        for(char &i:s){
            v[i-'a']++;
        }
        for(char&i: t){
            v[i-'a']--;
        }
        for(int&i:v)if(i!=0)return false;
        return true;
    }
};

File: Data Structures & Algorithms/duplicate-integer/submission-11.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
    #    hashset = []
    #    for num in nums:
    #       if num in hashset:
    #         return True
    #       hashset.append(num)

    #    return False
         
        hashset = []
        for num in nums:
            if num in hashset:
                return True
            hashset.append(num)

        return False

